lib/get_state_manager/src/simple/get_controllers.dart

```diff
@@ -109,28 +273,84 @@ mixin FullLifeCycleMixin on FullLifeCycleController {
   @mustCallSuper
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
-    switch (state) {
-      case AppLifecycleState.resumed:
-        onResumed();
-        break;
-      case AppLifecycleState.inactive:
-        onInactive();
-        break;
-      case AppLifecycleState.paused:
-        onPaused();
-        break;
-      case AppLifecycleState.detached:
-        onDetached();
-        break;
-      case AppLifecycleState.hidden:
-        onHidden();
-        break;
+    try {
```

Comment:
Here the error would not be coming from the library code, but from within some function in the user code. This could obfuscate the debugging. Unless we use something like runZoned (which would be overkill), this is unnecessary in my opinion.